node/src/BaseClient.ts

```diff
@@ -1316,6 +1317,31 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Gets the union of all the given sets.
+     *
+     * See https://valkey.io/commands/sunion/ for more details.
+     *
+     * @remarks When in cluster mode, all `keys` must map to the same hash slot.
+     * @param keys - The keys of the sets.
+     * @returns A set of members which are present in at least one of the given sets.
```

Comment:
Huh, looks like some commands have a "-" after the @returns and some don't.

Reply:
it would be better if they were consistent.  But javadoc in VSCode adds the dash if it's missing:
<img width="911" alt="image" src="https://github.com/valkey-io/valkey-glide/assets/104530826/848cae22-39fa-465f-9127-ca946e4c990f">

Reply:
I found that VS code can render docs with and without dashes. I don't know the standard.
Agree, that we need to align that. Can you create a task please?

Reply:
Yep, created a task on our board